ui/partials/main/thJobDetailsRetriggerMenu.html

```diff
@@ -1,6 +1,6 @@
 <li>
-  <a ng-click="retriggerJob([selectedJob])">Retrigger job</a>
+  <a ng-click="retriggerJob([selectedJob])" title="Re-run this job on this push">Retrigger job</a>
```

Comment:
How about `Repeat the selected job` (I think this is clear enough - "repeat" means do the same thing again and backfill's tooltop will have more information on the other case anyway)